pkg/minikube/out/out.go

```diff
@@ -141,7 +141,7 @@ func BoxedWithConfig(cfg box.Config, st style.Enum, title string, text string, a
 	}
 	// need to make sure no newlines are in the title otherwise box-cli-maker panics
 	title = strings.ReplaceAll(title, "\n", "")
-	boxedCommon(String, cfg, title, text, a...)
+	boxedCommon(Stringf, cfg, title, text, a...)
```

Comment:
as above: not sure if this is correct

Reply:
I renamed the `String` func to `Stringf` but none of the logic changed, so this should be identical to before.